samples/helloworld-agent-kotlin/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.akka</groupId>
+    <artifactId>akka-javasdk-parent</artifactId>
+    <version>3.5.2</version>
+  </parent>
+
+  <groupId>com.example</groupId>
+  <artifactId>helloworld-agent-kotlin</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>helloworld-agent-kotlin</name>
+
+  <properties>
+    <kotlin.version>2.2.10</kotlin.version>
+  </properties>
+
+
+  <dependencies>
+    <!-- Your dependencies go here -->
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib-jdk8</artifactId>
+      <version>${kotlin.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-test</artifactId>
+      <version>${kotlin.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+
+  <build>
+    <plugins>
+      <plugin>
+        <!-- Configures maven build to compile Kotlin classes and trigger Akka's annotation
+        processor -->
+        <groupId>org.jetbrains.kotlin</groupId>
+        <artifactId>kotlin-maven-plugin</artifactId>
```

Comment:
We should consider moving it under the parent pom. 

It doesn't have much impact on the build in the sense that if there is no kotlin class, the compiler is skipped. 

There is not much setup here, but still the setup is special because of our own parent pom, some extra flags we need for the compiler and the annotation processor. 

Having it in the parent will gives us more control over its evolution and allow us to 'update' users build whenever we need.